monkey/infection_monkey/credential_collectors/ssh_collector/ssh_handler.py

```diff
@@ -101,6 +107,16 @@ def _get_ssh_files(
                                                     ScanStatus.USED, info["name"], info["home_dir"]
                                                 )
                                             )
+
+                                            collected_credentials = Credentials(
+                                                identity=Username(info["name"]),
+                                                secret=SSHKeypair(
+                                                    info["private_key"], info["public_key"]
+                                                ),
+                                            )
+                                            _publish_credentials_stolen_event(
+                                                collected_credentials, event_queue
+                                            )
```

Comment:
I'm not sure it makes sense to call this in a function called `_get_ssh_files()`. Why don't we just publish the event in `SSHCredentialCollector.collect_credentials`?